apps/test/unit/templates/levelSummary/SummaryResponsesTest.jsx

```diff
@@ -51,53 +49,27 @@ const setUpWrapper = (state = {}, jsData = {}) => {
     {...INITIAL_STATE, ...state}
   );
 
-  const wrapper = mount(
+  return render(
     <Provider store={store}>
       <SummaryResponses scriptData={{...JS_DATA, ...jsData}} />
     </Provider>
   );
-
-  return wrapper;
 };
 
 describe('SummaryResponses', () => {
   it('renders elements', () => {
-    const wrapper = setUpWrapper();
-
-    // Student responses.
-    expect(wrapper.find(`.${styles.studentsSubmittedRight}`).text()).toBe(
-      '1/1 students answered'
-    );
-    expect(wrapper.find(`div.${styles.studentAnswer}`).length).toBe(1);
-    // Section selector, with one section.
-    expect(wrapper.find('SectionSelector').length).toBe(1);
-    expect(wrapper.find('SectionSelector option').length).toBe(2);
-  });
-
-  it('applies correct classes when rtl', () => {
-    const wrapper = setUpWrapper({
-      isRtl: true,
-      progress: {
-        currentLessonId: 0,
-        currentLevelId: '0',
-        lessons: [
-          {
-            id: 0,
-            levels: [
-              {activeId: '0', position: 1},
-              {activeId: '1', position: 2},
-            ],
-          },
-        ],
-      },
-    });
-
-    expect(wrapper.find(`.${styles.studentsSubmittedRight}`).length).toBe(0);
-    expect(wrapper.find(`.${styles.studentsSubmittedLeft}`).length).toBe(1);
+    renderWithProviders();
```

Comment:
Nit: in general, we've been using `renderDefault` for the setup methods, even if they setup the reducers and redux.